common/model-views.cpp

```diff
@@ -560,7 +580,8 @@ namespace rs2
             if (opt == RS2_OPTION_HOLES_FILL)
                 use_option_name = false;
 
-            auto desc = endpoint->get_option_description(opt);
+            auto desc_str = get_option_description();          
```

Comment:
Pass `opt` as a param for clarity

Reply:
No problem for me, but this is not needed at all, since this is not a static method, but a private one. 
Do you prefer it to be a static method?

Reply:
Leave it as a private, but pass it as a param.
Also, in the unexpected event of multi-thread call, the may change while the call is executed

Reply:
OK